lib/shopify-cli/tasks.rb

```diff
@@ -12,15 +12,22 @@ def add(const, name)
       end
 
       def [](name)
-        @tasks[name]
+        class_or_proc = @tasks[name]
+        if class_or_proc.is_a?(Class)
+          class_or_proc
+        elsif class_or_proc.respond_to?(:call)
+          class_or_proc.call
+        else
+          class_or_proc
```

Comment:
Can we simplify this branch, since we return `class_or_proc` for anything that's not callable?